packages/sample-app/webpack.config.babel.js

```diff
@@ -35,32 +34,30 @@ const developmentConfig = () =>
       // Increase performance budget thresholds for development mode
       maxAssetSize: 1500000,
       maxEntrypointSize: 1500000
-    })
+    }),
+    css.modules()
   ])
 
 const productionConfig = () =>
   group([
-    extractText(),
-    addPlugins([
-      new webpack.LoaderOptionsPlugin({
-        minimize: true,
-        debug: false
-      })
-    ])
+    css.modules({ minimize: true }),
+    extractText('css/[name].[contenthash:hex:8].css')
```

Comment:
Had to change it in order to fix the [`contenthash` issue](https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/763).